src/Features/CSharp/Portable/InvertIf/CSharpInvertIfCodeRefactoringProvider.cs

```diff
@@ -172,7 +174,8 @@ protected override StatementSyntax AsEmbeddedStatement(IEnumerable<StatementSynt
             IfStatementSyntax ifNode,
             SyntaxNode condition,
             StatementSyntax trueStatement,
-            StatementSyntax falseStatementOpt = null)
+            StatementSyntax falseStatementOpt = null,
+            bool shouldAddElasticTrivia = false)
```

Comment:
fwiw, i don't like optional args in internal apis (esp. feature=specific apis) as they are too easy to miss updating necessary callers).